editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -819,7 +857,11 @@ export const removeNodesDependentOnPassport = (
   breadcrumbsWithoutPassportData: Store.breadcrumbs;
   removedNodeIds: string[];
 } => {
-  const DEPENDENT_TYPES = [TYPES.PlanningConstraints, TYPES.DrawBoundary];
+  const DEPENDENT_TYPES = [
+    TYPES.PlanningConstraints,
+    TYPES.DrawBoundary,
+    TYPES.PropertyInformation,
+  ];
```

Comment:
This says that "PropertyInformation" should be cleared & shown fresh anytime "FindProperty" changes, rather than retain previously submitted data like other components do when navigating backwards/forwards.